### ISSUE TITLE
Inconsistent references to EventContent in /sync spec

### ISSUE THREAD

**Issue by joepie91** [matrix-org/matrix-doc#2274]
In various places in the /sync endpoint spec for the CS API, `EventContent` is specified as the type for fields, but in other places it just says `object`, even though they all contain the same kind of thing (namely, varyingly-structured event content) and `EventContent` is not defined for that endpoint.

**Comment by DMRobertson** [comment]
I found this confusing today when looking at `/sendToDevice`.

Maybe the intention is that each different kind of Event defines its own `EventContent` type, and if unspecified it's assumed to be an arbitrary json object (i.e. a map)? But that doesn't seem to match how `UnsignedData` works---I think it always contains the same set of three keys everywhere.

Hyperlinks between named types would be great if possible.

**Comment by uhoreg** [comment]
Yes, each type of event defines its own content.  Maybe we should name each type differently (e.g. `InviteEventContent`, `MemberEventContent`, etc)
